docs/ldml/tr35-keyboards.md

```diff
@@ -1348,206 +1343,503 @@ This example shows some of the mappings for a French keyboard layout:
 
 * * *
 
+### Element: variables
+
+> <small>
+>
+> Parents: [keyboard](#Element_keyboard)
+>
+> Children: [import](#Element_import), [_special_](tr35.md#special), [variable](#Element_variable)
+>
+> Occurrence: ?, ?
+> </small>
+
+* * *
+
+### <a name="Element_variables" href="#Element_variables">Element: variables</a>
```

Comment:
Why are there two separate versions of this header?  The first version is incredibly unclear in purpose, looking like a shell that led to the second one.

Reply:
typo ,fixed thanks